build.sbt

```diff
@@ -75,3 +70,15 @@ lazy val checkerPlugin = (projectMatrix in file("checker-plugin"))
   .settings(libraryDependencies ++= Seq(akkaTyped % Test, akkaPersistence % Test, akkaProjections % Test))
   .dependsOn(checkerLibrary)
   .jvmPlatform(scalaVersions = supportedScalaVersions)
+
+lazy val schemaDumpPlugin = (projectMatrix in file("sbt-dumpschema"))
+  .enablePlugins(SbtPlugin)
+  .settings(name := "sbt-dumpschema")
+  .settings(commonSettings)
+  .settings(pluginCrossBuild / sbtVersion :=
+    virtualAxes.value
+      .collectFirst { case x: ScalaVersionAxis => x.value }
+      .map { case "2.12" => "1.2.8" }
```

Comment:
Hmmm actually... could you provide a link on how you decided on which sbt version should correspond to the given Scala version?

So `sbt` up to 1.4 (incl.) is based on scala 2.12, and only sbt 1.5 is based on scala 2.13, right?

Reply:
We need a feature from sbt 1.2 that allows for `.enablePlugins(SbtPlugin)`
The whole sbt 1.x family runs on 2.12.

Reply:
Hmmm okay... so why the special case for `1.5.2`? 🤔

Reply:
In the newest version it's always `1.2.8`

Reply:
Ok... so the `1.5.2` case is just added defensively, we won't ever fall into that case in reality?

Reply:
If so, pls add a comment

Reply:
Although probably just doing a `.get` would be better, if Scala version is NOT 2.12 then the build will crash (rather than silently proceed with untested settings)

Reply:
This fragment of code looks differently in the newest revision, so I think this discussion no longer applies
```scala
.settings(pluginCrossBuild / sbtVersion := "1.2.8")
.dependsOn(schemaDumpCompilerPlugin)
.jvmPlatform(scalaVersions = Seq(scalaVersion212))
```